src/definitions/dokan-core.ts

```
@@ -0,0 +1,7 @@
+import { User } from '@wordpress/core-data';
+
+export type CoreState = {
+    currentUser: User;
+    store: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)